src/test/CMakeLists.txt

```diff
@@ -527,18 +527,18 @@ if(WITH_FIO OR WITH_SYSTEM_FIO)
 endif()
 
 if(WITH_RBD)
-  add_ceph_test(run-rbd-unit-tests.sh ${CMAKE_CURRENT_SOURCE_DIR}/run-rbd-unit-tests.sh)
+  add_ceph_test(run-rbd-unit-tests.sh ${CMAKE_CURRENT_SOURCE_DIR}/run-rbd-unit-tests.sh LABELS "long" COST 1000.0)
   if(FREEBSD)
-    add_ceph_test(rbd-ggate.sh ${CMAKE_CURRENT_SOURCE_DIR}/rbd-ggate.sh)
+    add_ceph_test(rbd-ggate.sh ${CMAKE_CURRENT_SOURCE_DIR}/rbd-ggate.sh LABELS "vstart;freebsd" COST 160.0)
```

Comment:
How are these specific figures obtained? Could you please document that somewhere (https://docs.ceph.com/docs/master/dev/developer_guide/#make-check)?

Reply:
I've taken the runtime of those identified cases from:
http://cephdev.digiware.nl:8180/jenkins/job/ceph-master/3720/console

I'll see if I can plug some comments about the extra options in the dev-docs.